packages/template-retail-react-app/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "1.1.0-dev",
```

Comment:
Just leaving to note that we are keeping `-dev.0` in packages released individually.
I think this is expected in this exceptional case, but in the future, we'll want to keep `-preview` and `-dev` consistent across all packages for consistency.